mapdata/WarcraftLegacies/AbilityData/Skin/925913172.json

```diff
@@ -0,0 +1,46 @@
+{
+  "OldId": 1952998209,
+  "NewId": 925913172,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": ""
+    },
+    {
+      "Id": 2037082209,
+      "Type": 3,
+      "Value": "Q"
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Slams the ground violently reducing enemy units movement speed by 75%, attack rate by 50%, and damage by 50% with an AOE of 700. |nLasts 15 seconds."
+    },
+    {
+      "Id": 1835101793,
+      "Type": 3,
+      "Value": "Tremor Stomp"
```

Comment:
Probably want to make the name reference Cripple if you can. `Crippling Stomp` maybe.